sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-files/SFAS-TO-SIMS-2024MAR07-LEGACY-RESTRICTIONS.txt

```diff
@@ -1,5 +1,5 @@
 100AESTSFAS TO SIMS BRIDGE                    20211119142437 
 2001010101010JOHN           FOUR                                    19980324900041310 0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000                                                            
-40010101010100964351111TREE20180927   
+40010101010100964351111SINR20180927   

```

Comment:
The expectation is that, the restriction that needs to be mapped to LGCY is SINF not SINR.

![image](https://github.com/user-attachments/assets/fc23fd25-479c-4548-9b61-42e46e099f7d)